src/Rx/Weno/file_download.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/**
+ * Weno Zipped file download
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Kofi Appiah <kkappiah@medsov.com>
+ * @copyright Copyright (c) 2023 omega systems group international <info@omegasystemsgroup.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../../../interface/globals.php");
+
+use OpenEMR\Common\Crypto\CryptoGen;
+use OpenEMR\Rx\Weno\LogDataInsert;
+use OpenEMR\Common\Logging\EventAuditLogger;
+
+$cryptogen = new CryptoGen();
+$weno_username = $GLOBALS['weno_admin_username'];
+$weno_password = $GLOBALS['weno_encryption_key'];
```

Comment:
note that the encrypted form of a value/password can be different even if the value/password does not change (i see that the encrypted value is being used below to build the MD5Password)